src/table/table.scss

```diff
@@ -16,75 +15,205 @@
+        width: $iui-xxs;
+        margin: 0 auto;
+        transition: background-color $iui-speed-fast ease-out, width $iui-speed-fast ease-out;
+        @include themed {
+          background-color: t(iui-color-background-5);
+        }
+      }
+
+      &:hover > .iui-resizer-bar {
+        width: $iui-xs;
+        @include themed {
+          background-color: t(iui-color-foreground-primary);
+        }
+      }
+    }
+
+    &:hover,
+    &:focus,
+    &:focus-within {
+      @include themed {
+        background-color: t(iui-color-background-4);
+      }
 
+      .iui-sort,
+      .iui-borderless {
+        visibility: visible;
+      }
+    }
+
+    &:hover {
+      > .iui-resizer {
+        display: block;
+      }
+    }
+  }
+
+  // Sort icon
+  .iui-sort {
+    visibility: hidden;
+    margin-left: auto;
+    width: $iui-icons-default;
+    height: $iui-icons-default;
     @include themed {
-      border-bottom: solid 1px t(iui-color-background-4);
+      fill: t(iui-icons-color-actionable);
     }
+  }
 
-    > .iui-tables-slot > .iui-tables-menu {
-      visibility: hidden;
+  // Sorted column
+  .iui-sorted {
+    @include themed {
+      background-color: t(iui-color-background-4);
+    }
+
+    .iui-sort {
+      visibility: visible;
+        
+      &,
+      > * {
+        @include themed {
+          fill: t(iui-icons-color-actionable);
+        }
+      }
+    }
+  }
+
+  // Double header
+  .iui-row + .iui-row {
+    @include themed {
+      border-top: 1px solid t(iui-color-background-5);
     }
+  }
+}
+
+@mixin iui-table-body {
+  width: 100%;
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+  
+  .iui-row {
+    display: flex;
+    flex-grow: 1;
+    width: 100%;
+    margin-top: -1px;
+    border: solid 1px transparent;
 
-    &:hover:not(.iui-tables-row-active, .iui-tables-paginator) {
+    // Last row doesn't get bottom border. Up for debate.
+    &:not(:last-child) {
       @include themed {
-        background-color: rgba(
-          t(iui-color-foreground-primary-rgb),
-          t(iui-opacity-6)
-        ) !important;
+        border-bottom: solid 1px t(iui-color-background-4);
+      }
+    }
+
+    > .iui-slot > .iui-more-options {
+      visibility: hidden;
+    }
+
+    &:hover:not(.iui-disabled) {
+      &:not(.iui-selected):not(.iui-expanded-content) {
+        @include themed {
+          background-color: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-6)
+          );
+        }
       }
 
-      > .iui-tables-slot > .iui-tables-menu {
+      > .iui-slot > .iui-more-options {
         visibility: visible;
       }
     }
 
-    > .iui-tables-cell {
+    &:focus-within > .iui-slot > .iui-more-options {
+      visibility: visible;
+    }
+
+    > .iui-cell {
+      position: relative;
       align-items: center;
       display: flex;
       flex-grow: 1;
-      min-width: 145px;
-      padding: $iui-table-density;
+      min-width: $iui-l * 6;
```

Comment:
Not sure about this one...we might have really little info in some columns, so width of 144px in normal table might be too big..

Reply:
I just did a test to see how macOS handles this.  All columns do have a min-width set, but it appears to be different for every column.  In order from left to right:
1. 172px
2. 70px
3. 74px
4. 75px

![Screen Shot 2021-06-01 at 3 58 38 PM](https://user-images.githubusercontent.com/849817/120382923-3b6ff800-c2f2-11eb-80ca-2be654e12734.png)

This is leading me to think that we do need a min-width set, but maybe we leave it up to the development team?

Reply:
In our case, first column is 48px which is the checkbox and every other column is at least 145px, though you can set it to 50px (58px with margin) which doesn't display anything useful. At this point, you should hide that column :
![image](https://user-images.githubusercontent.com/4070713/120383792-5131ed00-c2f3-11eb-841f-ba9291a788cb.png)

Reply:
You might want to have less than 145px for sure!
For example, index column:
![image](https://user-images.githubusercontent.com/81580355/120647830-b0266c00-c483-11eb-9422-c376c25948a4.png)

Reply:
I changed to 64px.